packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -133,7 +135,7 @@ export const CardHeader: React.FunctionComponent<CardHeaderProps> = ({
 
         const SelectableCardInput = selectableActions?.variant === 'single' ? Radio : Checkbox;
         const getSelectableProps = () => ({
-          className: 'pf-m-standalone',
+          className: css('pf-m-standalone', selectableActions?.isHidden && 'pf-v6-screen-reader'),
```

Comment:
Could you use an object from react-styles for the screen reader class so that we don't have to manually bump it to v7 etc in the future?

Reply:
I did not think there was an object for this, but I found it in `@patternfly/react-styles/css/utilities/Accessibility`
I will open tech debt issue to replace the other instances of this string in our code base.

Reply:
https://github.com/patternfly/patternfly-react/issues/10955